Samples/ios/SampleApp/SampleAppiOS/SampleMSALUtil.m

```diff
@@ -0,0 +1,217 @@
+//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#import <MSAL/MSAL.h>
+
+#import "SampleMSALUtil.h"
+#import "SampleAppErrors.h"
+#import "SamplePhotoUtil.h"
+
+#define CURRENT_USER_KEY @"MSALCurrentUserIdentifier"
+
+@implementation SampleMSALUtil
+
++ (void)setup
+{
+    [[MSALLogger sharedLogger] setCallback:^(MSALLogLevel level, NSString *message, BOOL containsPII)
+    {
+        // Log messages are played in both comtainsPII and not containsPII versions, so we only need
+        // to capture one of them.
+        if (containsPII)
+        {
+            NSLog(@"%@", message);
+        }
+    }];
+}
+
++ (MSALPublicClientApplication *)clientApplication
+{
+    // This MSALPublicClientApplication object is the representation of your app listing, in MSAL. For your own app
+    // go to the Microsoft App Portal (TODO: Name? Link?) to register your own applications with their own client
+    // IDs.
+    return [[MSALPublicClientApplication alloc] initWithClientId:@"0615b6ca-88d4-4884-8729-b178178f7c27" error:nil];
```

Comment:
do we typically put actual client ids into samples? perhaps just replace with ???